app/services/payout_report_publisher_includer.rb

```diff
@@ -10,40 +10,52 @@ def perform
     publisher_has_unsettled_balance = false
 
     wallet = @publisher.wallet
-    # If the wallet has been blocked from uphold's terms of service. In this state users are unable to login or access uphold.
-    return if wallet.nil? || wallet.blocked?
```

Comment:
The wallet should never be nil, and we want to record if it's blocked, so removing this line entirely